CMakeLists.txt

```
@@ -56,6 +56,9 @@ endif()
 
 if (BUILD_WITH_PCL)
   find_package(PCL REQUIRED)
+  if(PCL_VERSION VERSION_LESS "1.11.0")
+    message(FATAL_ERROR "The found PCL version ${PCL_VERSION} is too low. Required is at least 1.11.0")
+  endif()
```

Can be simplified to just
```suggestion
  find_package(PCL 1.11.0 REQUIRED)
```

---

So nice, @valgur, but if so, what if one uses PCL 1.12. or more higher version?
Plus, it's my personal question, but I'm curious whether it is possible to install multiple PCL versions on a machine. Thx!

---

@LimHyungTae 
Thanks for your contribution. The way suggested by @valgur only specifies the minimum required version and should accept newer versions.